.travis.yml

```diff
@@ -1,37 +1,40 @@
 sudo: false
 language: python
 python:
+- '2.6'
+- '2.7'
+- '3.3'
+- '3.4'
 - '3.5'
+- nightly
+
+cache: pip
 install: pip install -U tox setuptools_scm
 env:
   matrix:
-  - TESTENV=py26-pytest26
```

Comment:
I'm curious, why did you change how the environment was declared? TBH I prefer the old way, because it is more explicit and can easily be updated by using the output from `tox -l`...